xml/deployment_configuration.xml

```diff
@@ -318,15 +320,109 @@ ssh_authorized_keys:
      will be prompted to change the default password at the first login.
     </para>
    </sect3>
-   <sect3 xml:id="user-data.config.adding.custom.repository">
+      <sect3 xml:id="user-data.config.adding.custom.repository">
     <title>Adding Custom Repository</title>
-    <para>
-     You can add a custom software repository to your system by using the
-     <literal>zypp_repos</literal> option:
-    </para>
-<screen>zypp_repos:
-  "Update Repo":
-      baseurl: http://my.example.com/CAASP/repo/SUSE-CAASP-1.0-CUSTOM/</screen>
+    <para> You can add a custom software repository to your system by using the
+      <literal>zypp_repos</literal> option: </para>
+    <screen>
+        zypper:
```

Comment:
Note that the content of the screen element is displayed 'as-is' including whitespaces, any indention in the source are present in the printed media. Don't indent something that shouldn't be. In this case, zypper shouldn't be indented, only the following lines should be relative to it.

Reply:
tl;dr: Unindent everything by 4 chars.